charmhelpers/contrib/hahelpers/cluster.py

```diff
@@ -361,3 +365,24 @@ def canonical_url(configs, vip_setting='vip'):
     else:
         addr = unit_get('private-address')
     return '%s://%s' % (scheme, addr)
+
+
+def distributed_wait(modulo=None, wait=None):
+    ''' Distribute operations by waiting based on modulo_distribution
+
+    If modulo and or wait are not set, check config_get for those values.
+
+    :side effect: Calls config_get()
+    :side effect: Calls log()
+    :side effect: Calls status_set()
+    :side effect: Calls time.sleep()
+    '''
```

Comment:
I think we should provide some :param: docstrings which explain the values a bit (even though its clear to me). Also, I think we need to be explicit with what the unit wait interval is specified as (seconds).

Reply:
Yep, that was an oversight.